integration/entrypoint.sh

```diff
@@ -77,6 +77,11 @@ function retry {
     fi
 }
 
+function can_erofs_ondemand_read {
+    grep CONFIG_EROFS_FS_ONDEMAND=y /usr/src/linux-headers-"$(uname -r)"/.config
```

Comment:
Can this test case run in github action? or just run in an internal host?

Reply:
Github action does not have a runner host with the kernel of that new (>=5.19). So it only runs locally